packages/react/src/Box/Box.features.module.css

```diff
@@ -0,0 +1,48 @@
+.spacingContainer {
```

Comment:
Are these styles just for documentation? I wonder if they should live in Storybook instead.

Reply:
👍 Those don't go into the user bundle, they're only imported into Storybook. Is that what you mean by living in Storybook?